### ISSUE TITLE
413 Request entity too large

### ISSUE THREAD

**Issue by CODEheures** [phpmyadmin/docker#447]
# With this configuration:
![2024-12-04_14h03_51](https://github.com/user-attachments/assets/83f73539-ca40-4838-8e21-3cf7a4982a8b)
![2024-12-04_14h14_59](https://github.com/user-attachments/assets/b685f7c6-7f39-4642-9dd4-6ca265af2401)

# I try to upload this file (~1Go):
![2024-12-04_14h12_50](https://github.com/user-attachments/assets/7fd4a5bc-a1e1-42de-9674-73f3d1f76282)

# And I obtain 413:
![2024-12-04_14h15_20](https://github.com/user-attachments/assets/39247d3a-606e-4ae8-851b-e23962923b91)

**Comment by williamdes** [comment]
Thank you for the report, I need to investigate.
That said you should be using `phpmyadmin` as the image name, it is the official Docker one. Ref #363

**Comment by CODEheures** [comment]
Thank you. I change to use the official. For info, I obtain the same result

![2024-12-09_10h41_39](https://github.com/user-attachments/assets/5348531a-6c38-4c0b-92fb-f4e018188c64)

**Comment by appmonkey8010** [comment]
Same Issue here. Only the latest Version. Downgrade to phpmyadmin/phpmyadmin:5.2.0 and work now. Seems to be, that the variables are not set corretly

**Comment by williamdes** [comment]
Here is the diff, this is quite strange: https://github.com/phpmyadmin/docker/compare/5.2.0...5.2.1.1

Can you use : `phpmyadmin:5.2.0 ` and confirm that it still works fine ?
But not `phpmyadmin:5.2.1`

**Comment by CODEheures** [comment]
I just did the test and I confirm that the upload work fine with `phpmyadmin:5.2.0` but not with `phpmyadmin:5.2.1`

**Comment by appmonkey8010** [comment]
Yes. Tested it. phpmyadmin:5.2.0 work but not with phpmyadmin:5.2.1

**Comment by williamdes** [comment]
Thanks for all the confirmations, I will work on this ASAP

**Comment by hauserkristof** [comment]
Same here with 5.2.1

**Comment by williamdes** [comment]
5.2.2 it building and should be out soon. I expect this bug to sill exist in 5.2.2

**Comment by CODEheures** [comment]
Sorry, I just tested with version 5.2.2 and the bug is still there

![Image](https://github.com/user-attachments/assets/5b6877c0-ca24-4d8b-9e5f-726ecd361aff)

![Image](https://github.com/user-attachments/assets/9d9fc5b4-4c1d-45f3-b940-a9b69d8a9818)

![Image](https://github.com/user-attachments/assets/733e0761-67e8-46f0-b563-1183eee02ab5)

**Comment by hauserkristof** [comment]
Any update on this?
